data/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.library'
+
+apply from: './gradleFiles/dependenciesData.gradle'
+
+android {
+    compileSdkVersion 28
```

Comment:
@OscarGonzalezLesaga Maybe we can set all this numeric values in an ext too, because when we change versions, we should only do it in one place

Reply:
Yeap, it's a good point. I'm going to create a 'config' file to set this values.